assets/sass/protocol/components/logos/_wordmark-product-vpn.scss

```diff
@@ -1,14 +1,14 @@
+/* stylelint-disable scss/dollar-variable-pattern */
```

Comment:
Nit: some of these comments are hard to spot when they're bunched right up to the licence block below. Suggest placing these on a separate line underneath the licence block for better readability?

(not a blocker)